docs/frontend_backend_run.md

```diff
@@ -134,7 +123,13 @@ Now that we have two applications running, let's make them communicate. By defau
     }
    ```
 
-4. Now register this client in `Startup.cs` class in `ConfigureServices` of the `frontend` project:
+4. Add a reference to the `Microsoft.Tye.Extensions.Configuration` package.
+
+    ```txt
+    dotnet add package Microsoft.Tye.Extensions.Configuration
```

Comment:
Do we need to add the feed until we ship to nuget?

Reply:
Yeah I need to do that in general. Now that we've got a library we need to include that in the instructions everywhere. Good catch.